lib/Command/CreateEmptyConfig.php

```diff
@@ -46,19 +47,39 @@ protected function configure() {
 		$this
 			->setName('ldap:create-empty-config')
 			->setDescription('creates an empty LDAP configuration')
+			->addArgument(
+				'configID',
+				InputArgument::OPTIONAL,
+				'create a configuration with the specified id'
+			)
 		;
 	}
 
 	protected function execute(InputInterface $input, OutputInterface $output) {
-		$configPrefix = $this->getNewConfigurationPrefix();
+		$availableConfigs = $this->helper->getServerConfigurationPrefixes();
+		$configID = $input->getArgument('configID');
+		if(is_null($configID)) {
+			$configPrefix = $this->getNewConfigurationPrefix($availableConfigs);
+		} else {
+			// Check we are not trying to create an empty configid
+			if($configID === '') {
+				$output->writeln("configID cannot be empty");
+				return 1;
+			}
+			// Check if we are not already using this configid
+			if(in_array($configID, $availableConfigs)) {
```

Comment:
The `in_array` function has a parameter to specify strict type checking. I'd rather use that option always by default or add a comment otherwise explaining why it isn't needed. This should prevent unwanted surprises.